rcli/config.py

```diff
@@ -104,6 +104,7 @@ def distribution(self):
         """The distribution containing the currently active entry point."""
         if self.entry_point:
             return self.entry_point.dist
+        return None
```

Comment:
Why the explicit return?

Reply:
pylint...   or flake8..  pylint I think.

Reply:
Are you running this with `tox`? That will make sure that you're running those with the correct settings.

Reply:
I am running tox, yes.